assets/website/javascriptLibraries.txt

```diff
@@ -0,0 +1,14 @@
+Javascript libraries
+====================
+
+Twitter bootstrap version: 3.1.1
+Downloaded from: http://twitter.github.com/bootstrap/
+License: MIT (which can be used by Apache License v2.0 projects)
+
+jQuery version: 1.11.0
+Downloaded from: http://jquery.com/
+License: MIT (which can be used by Apache License v2.0 projects)
```

Comment:
These are no longer the correct versions, nor are they being downloaded and served from this host.

Reply:
updated reflecting from:

- https://github.com/kiegroup/optaplanner-website/pull/406